Server/Test_Plan_Templateeditor.md

```diff
@@ -0,0 +1,22 @@
+## Templateeditor Test Plan
+
+
+
+| Test Case                                | Expected Result                          | Result         | Related Comment |
+| ---------------------------------------- | ---------------------------------------- | -------------- | --------------- |
+| Enable templateeditor.                   | App is enabled and can be used in the admin settings. | 🚧             |                 |
```

Comment:
It should be possible to enable the template editor via CLI. We must add an scenario